es-ds-docs/pages/atoms/corners.vue

```diff
@@ -1,5 +1,130 @@
 <template>
     <div>
-        atoms - corners
+        <h1>Corners</h1>
+        <p>
+            Below are the four corner radius sizes used throughout the design system. Each size lists a few examples of
+            components that make use of that size. Use the utility classes listed below (e.g. <code>rounded</code>) to
+            add the desired corner radius to your UI elements.
+        </p>
+
+        <b-row class="my-500">
+            <b-col
+                v-for="item in borderRadius"
+                :key="item.name"
+                class="mb-200"
+                cols="6"
+                lg="3">
+                <div
+                    class="bg-soft-blue h-100 px-100 px-lg-200 py-200 py-lg-300 text-center text-dark-blue"
+                    :class="item.class">
+                    <p class="font-size-300 mb-25">
+                        {{ `${item.name} / ${item.sizePx}px` }}
+                    </p>
+                    <p>
+                        <code class="text-body">
+                            {{ item.class }}
+                        </code>
+                    </p>
+                    <p class="font-size-sm mb-0">
+                        <ul class="component-list list-unstyled mb-0">
+                            <li class="d-inline-block">
+                                Used by
+                            </li>
```

Comment:
Though this code is the same as the old docs site, I'm seeing no space between the "Used by" text and the next item. Maybe we can throw an `&nbsp;` in here, or worst case, a `mr-25` class to add some space?

<img width="300" alt="Screen Shot 2024-08-01 at 4 53 37 PM" src="https://github.com/user-attachments/assets/28339b6f-dc9f-4b89-baad-d56f3c03ea98">

Reply:
Yeah, after examining the code I don't why that's happening. I tried adding `&nbsp;` and I'll put it back in.